charts/portworx/values.yaml

```diff
@@ -58,3 +58,13 @@ serviceAccount:
   hook:
     create: true
     name:
+
+api:
+  enabled: true                     # Deploy portworx-api daemonset.
+
+dynamicProvision:
+  enabled: false                    # Allow dynamic provisioning of persistent volumes from persistent volume claims. Requires api.enable to be true
```

Comment:
Dynamic provisioning of PVCs is the primary use case for Portworx. So this is always enabled. I don't see a use case of having this flag.